Hudl.Mjolnir/Command/Command.cs

```diff
@@ -380,9 +380,17 @@ private Task<TResult> ExecuteInIsolation(CancellationToken cancellationToken)
                 CircuitBreaker.MarkSuccess(stopwatch.ElapsedMilliseconds);
                 CircuitBreaker.Metrics.MarkCommandSuccess();
             }
-            catch (Exception)
+            catch (Exception e)
             {
-                CircuitBreaker.Metrics.MarkCommandFailure();
+                if (CommandContext.IsExceptionIgnored(e.GetType()))
```

Comment:
Could e be wrapping another exception let us the implements is awaiting on an async call for example, and that throws an exception, for now do we just get e's type? I don't see an immediate need but wanted to plant a seed, if it is worth planting